data-source/jsonnet/northern-ireland/household/blocks/who-lives-here/visitor_list_collector.jsonnet

```diff
@@ -100,6 +100,10 @@ function(census_date) {
   add_block: {
     id: 'add-visitor',
     type: 'ListAddQuestion',
+    cancel_text: {
+      text: 'Don’t need to add anyone else?',
+      action_text: 'Return to the previous page',
+    },
```

Comment:
I don't think there is a requirement for the action text to be configurable. We should just have `cancel_text` and if that's not in the schema no link should be displayed. The current fallback to `Cancel and return to the previous question` doesn't work as the previous page could be a summary.

Reply:
I split it out to two separate strings because of the requirement for census that the hyperlink would only be displayed on the 'action' part of text. I see what you mean about the fall back.

Reply:
This has been fixed.